### ISSUE TITLE
var can not be resolved to a type

### ISSUE THREAD

**Issue by kyla963** [tinderjs/tinderjs#37]
Hello, I am new to programming and this is my first time with java or eclipse. When I use the provided code for the bot I get an error on every line, most notably "var can not be resolved to a type". Obviously this is too many errors to be an issue with the code, so it has to be something I am doing wrong. I've looked online but can't find anything. What am I missing?

**Comment by mrmayfield** [comment]
Yup, I can help you, but show me the steps you are taking and the exact error message you receive. The more info you have give me the better.

**Comment by mrmayfield** [comment]
Haha, this is a Javascript library, not a Java library. So the syntax is all wrong with what you're trying. :)

**Comment by kyla963** [comment]
Oh. I get it. I'm an idiot. And this software is fantastic. Thank you for figuring that out in record time. I will google JavaScript set up. Again, much appreciated. It may have taken me days to figure that out.

**Comment by mrmayfield** [comment]
Haha no problem. Yeah get your feet wet with Javascript, there's a couple more concerns with your code but once you're using Javascript, post your code again and I'll help you out as I can 👍
